### PR TITLE
Update gRPC example for insecure host

### PR DESCRIPTION
**Pull request by kflynn**
(cherry picked from commit b602adf80a52ca85541ca4050dd58a7a5786ad4e)

### CODE REVIEW

File: docs/howtos/grpc.md [hunk 1]
```diff
@@ -141,7 +153,7 @@ spec:
       restartPolicy: Always
 ```
 
-After adding the Ambassador Edge Stack mapping to the service, the rest of the Kubernetes deployment YAML file is pretty straightforward. We need to identify the container image to use, expose the `containerPort` to listen on the same port the Docker container is listening on, and map the service port (80) to the container port (50051).
+The Host is declared here because we are using gRPC without TLS.  Since Ambassador terminates TLS by default, in the Host we add a `requestPolicy` which allows insecure connections . After adding the Ambassador Edge Stack mapping to the service, the rest of the Kubernetes deployment YAML file is pretty straightforward. We need to identify the container image to use, expose the `containerPort` to listen on the same port the Docker container is listening on, and map the service port (80) to the container port (50051).
```
Comment (khussey) [inline docs/howtos/grpc.md:156]: Extraneous space before period here...

**Review by khussey — CHANGES_REQUESTED** [review]
Minor nit.

File: docs/howtos/grpc.md (hunk 1 above)
Comment (kflynn) [reply docs/howtos/grpc.md]: Oops. Thanks! (Sorry, I missed this comment for awhile. 🙁 )

**Review by khussey — APPROVED** [review]
+1

**Pull request merged**